### DISCUSSION TITLE
Community suggested to use the Tailwindcss CDN to simplify my project

### DISCUSSION THREAD

**Discussion by Arvindjitta** [EddieHubCommunity/support#91] [Q&A] [answered]
Hello brother , as you said I've used cdn and working on my website it's going great :heart_eyes:.. but how do I share the website link with you ?? Bot is not allowing me to share links , my website link : https://arvindjitta.com/

**Comment by eddiejaoude** [comment, marked as answer]
Where is the GitHub url? 🤓

**Reply by Arvindjitta** [reply]
this is my website url https://arvindjitta.com/ ...what github url??

**Reply by eddiejaoude** [reply]
We are coders, we want to see code 😉

**Reply by eddiejaoude** [reply]
Best to post on GitHub before hosting it 🤓

**Reply by eddiejaoude** [reply]
The design looks good 👍

**Reply by eddiejaoude** [reply]
Follow this guide to put your project on GitHub https://docs.github.com/en/github/getting-started-with-github/create-a-repo

**Reply by Arvindjitta** [reply]
> The design looks good 👍

Thank you, that means a lot <3
